File: TransCoder_tokenized_test_set_functions/java/KTH_ADJACENT_NODE_GRAPH_VERTEX_WEIGHT.java
static void printkthnode ( Vector < pair > adj [ ], int wt [ ], int n, int k ) {
  for ( int i = 0;
  i < n;
  i ++ ) Collections . sort ( adj [ i ], new Comparator < pair > ( ) {
    public int compare ( pair p1, pair p2 ) {
      return p1 . first - p2 . first;
    }
  }
  );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( adj [ i ] . size ( ) >= k ) System . out . print ( adj [ i ] . get ( adj [ i ] . size ( ) - k ) . second + " " );
    else System . out . print ( "-1" );
  }
}

File: TransCoder_tokenized_test_set_functions/python/KTH_ADJACENT_NODE_GRAPH_VERTEX_WEIGHT.py
def printkthnode ( adj, wt, n, k ) :
    for i in range ( n ) :
        adj [ i ] . sort ( )
    for i in range ( n ) :
        if ( len ( adj [ i ] ) >= k ) :
            print ( adj [ i ] [ len ( adj [ i ] ) - k ] [ 1 ], end = " " )
        else :
            print ( "-1", end = " " )
    
